src/main/scala/za/co/absa/commons/os/OperatingSystem.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.commons.os
```

Comment:
Do you think it deserves a separate package? Shouldn't we just place in into the `lang` instead?

Reply:
Yes, I am favoring a separate approach on this, but I can be persuaded if this is unwanted.

Reply:
The stuff that we put into the Commons, from the view of semantics, usage and dependencies basically form clusters. 
The stuff like unit-test framework extensions, web utils, graph utils etc could potentially be seen as separate libraries. The main reason why they are still in one library is because of there are too few of them and we weren't sure how to break them down into sub-projects without risking to make it too fine or too course granular. So, for the time being we decided to keep stuff in one project, make all dependencies optional and make the best effort to keep the package structure resembling the sub-project break down. So that eventually, when there is more "meat" in the project it would split into separate jars and released independently, similarly to e.g. Apache Commons. The other logical outcome of this is to keep the number of top-level packages reasonably low.
The package `lang` contains stuff that do not require any additional dependency and contains very generic stuff that semantically close to the language/platform itself than any other library or a task specific domain.
And since this "OS" stuff both is generic and doesn't require any dependency I would suggest to put it into the `lang` package.

Reply:
Thanks for the explanation, makes sense. Persuaded. I have moved the code into the `lang` package. 👍